src/content/docs/design/design-with-code/manage-design-assets.mdx

```diff
@@ -0,0 +1,90 @@
+---
+page_id: 14aeb42e-8799-481c-88cf-62f521b33d5e
+title: Manage page design assets
+sidebar:
+  order: 6
+relatedArticles:
+  - f0bc688b-a817-42ab-9a20-8e09cec06f37
+  - 66f83c18-d261-48b8-a517-648ed1b7b064
+---
+
+It is likely that your logos and favicons are hosted on Kinde, but you may also manage extrernally hosted assets. Here's how assets are referenced in design code.
```

Comment:
@clairekinde11 `extrernally` typo

Reply:
@clairekinde11 `Here's how assets are referenced in your code.` rather than design code